packages/c/cppp-reiconv/xmake.lua

```diff
@@ -0,0 +1,71 @@
+package("cppp-reiconv")
+    add_deps("cmake", "python")
+    set_homepage("https://github.com/cppp-project/cppp-reiconv")
+    set_description("A character set conversion library based on GNU LIBICONV.")
+
+    add_urls("https://github.com/cppp-project/cppp-reiconv/releases/download/$(version)/cppp-reiconv-$(version).zip")
+
+    add_versions("v2.1.0", "3e539785a437843793c5ce2f8a72cb08f2b543cba11635b06db25cfc6d9cc3a4")
+
+    add_deps("cmake")
+    
+    on_download(function (package, opt)
```

Comment:
> on_download? our package need source pakcage to build, origin git-repo can't build. ====================== 移除on_download？我们的包需要源码包来编译，原始的git仓库是不能正常编译的。

please remove it, xmake support download and extract tar package. you can refer other packages without git url.

https://github.com/xmake-io/xmake-repo/blob/42758138c5b4231f08e286a98111aafe2d8a2fb6/packages/p/pupnp/xmake.lua#L5

Reply:
> If we use original git tree to build, it will need some extra dependencies like 'git', 'bash' or 'cmd.exe' and 'GNU Make' etc. Use tarball to build is easier and faster. ================================ 如果真的用git文件树生成的话，需要额外的一些依赖如'git', 'bash'或'cmd.exe' 和 'GNU Make' 等等。使用源码包来编译更简单还更快。

you can use tarball to build it, but you need also to remove it.

Reply:
ok